ballet/eng/misc.py

```diff
@@ -1,18 +1,43 @@
 import funcy
 import numpy as np
 import pandas as pd
+from scipy.special import boxcox1p
+from scipy.stats import skew
 
 from ballet.eng.base import BaseTransformer, SimpleFunctionTransformer
 from ballet.util import get_arr_desc
 
-__all__ = ['IdentityTransformer', 'ValueReplacer', 'NamedFramer']
+__all__ = ['IdentityTransformer', 'BoxCoxTransformer', 'ValueReplacer', 'NamedFramer']
 
 
 class IdentityTransformer(SimpleFunctionTransformer):
     def __init__(self):
         super().__init__(funcy.identity)
 
 
+class BoxCoxTransformer(BaseTransformer):
+    def __init__(self, threshold, lmbda=0):
+        super().__init__()
+        self.threshold = threshold
+        self.lmbda = lmbda
+
+    def fit(self, X, y=None, **fit_args):
+        self.features_to_transform_ = abs(skew(X)) > self.threshold
+        if (isinstance(X, pd.DataFrame)):
+            # Hack to get a mask over columns
+            self.features_to_transform_ = X.columns.putmask(~self.features_to_transform_, None) & X.columns
+        return self
+
+    def transform(self, X, **transform_args):
+        if isinstance(X, pd.DataFrame):
```

Comment:
use `sklearn.utils.validation.check_is_fitted` to make sure `fit` has been called. see https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/preprocessing/data.py for examples.
```suggestion
        check_is_fitted(self, 'features_to_transform_')
        if isinstance(X, pd.DataFrame):
```

Reply:
This has been added!